src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -236,7 +236,9 @@ export class ExtensionsListView extends ViewletPanel {
 
 			const result = local
 				.sort((e1, e2) => e1.displayName.localeCompare(e2.displayName))
-				.filter(e => runningExtensions.every(r => !areSameExtensions(r, e)) && (e.name.toLowerCase().indexOf(value) > -1 || e.displayName.toLowerCase().indexOf(value) > -1));
+				.filter(e => e.type === LocalExtensionType.User	// exclude disabled builtin extensions
```

Comment:
If we ignore the built-in extensions here, then the only way one can get to them is to go the built-in view
Any reason for not including them here?

Reply:
Let's include the built-in disabled ones here.
Even today, when you go to the disabled extensions view, the disabled built in ones do appear there. So let's maintain that